src/docs/sdk/performance/ui-event-transactions.mdx

```diff
@@ -85,24 +85,24 @@ Scenario: Don't overwrite existing status of UI event transactions
     Then it keeps the status 
     And doesn't overwrite it
 
-Scenario: Same UI element with same event
-    Given an ongoing UI event transaction
-    When the user triggers the same UI event with the same type for the same
-        UI element
-    Then the SDK reschedule the idle timeout
-    And doesn't create a new transaction
-
 # If your SDK binds auto-generated transactions to the scope see binding
 # to scope
-Scenario: Same UI element with different event
-    Given an ongoing UI event transaction
-    When the user triggers the same UI element with a different event
+Scenario: New event on same UI element
+    Given an ongoing UI event transaction with child spans
+    When the user triggers the same UI element with a new event (different or same)
     Or the user triggers a different UI element
```

Comment:
`l`: There is already `Or the user triggers a different UI element` below so we don't need to mention it twice.
```suggestion
    When the user triggers the same UI element with a new event
    Or the user triggers a different UI element
```